src/Umbraco.Web/Scheduling/ScheduledTasks.cs

```diff
@@ -17,11 +17,12 @@ namespace Umbraco.Web.Scheduling
 
     internal class ScheduledTasks : RecurringTaskBase
     {
+        private static readonly HttpClient HttpClient = new HttpClient();
```

Comment:
Same as the earlier comments, it's only used in `GetTaskByHttpAync` so let's only initialize it there.